.gitignore

```diff
@@ -37,4 +37,6 @@ _ignored/
 
 # MacOS
 .DS_Store
-.idea
\ No newline at end of file
+.idea
+
+samconfig.yaml
```

Comment:
Ah.. now I know why it was checked in in the first place.  
Isn't this ususally a `.toml` file?

Reply:
What do you mean? I am not sure if I got your point